apps/etterlatte-behandling/src/main/kotlin/inntektsjustering/AarligInntektsjusteringJobbService.kt

```diff
@@ -76,13 +78,46 @@ class AarligInntektsjusteringJobbService(
 ) {
     private val logger = LoggerFactory.getLogger(this::class.java)
 
-    fun startAarligInntektsjustering(request: AarligInntektsjusteringRequest) {
+    // for å starte inntektsjusteringJobb for alle saker som er løpende i neste inntektsår
+    fun startAarligInntektsjusteringJobb(request: AarligInntektsjusteringRequest) {
+        logger.info("Starter årlig inntektsjusteringjobb $kjoering")
         request.saker.forEach { sakId ->
             startEnkeltSak(request.kjoering, request.loependeFom, sakId)
         }
     }
 
-    fun opprettManuellInntektsjustering(
+    // for å behandle inntektsjustering via innsendt skjema
+    fun startInntektsjusteringJobb(request: InntektsjusteringRequest) {
+        logger.info("Starter inntektsjusteringjobb for sak ${request.sak.sakId}")
+        val skalGjoeresAutomatisk =
+            featureToggleService.isEnabled(
+                InntektsjusterinFeatureToggle.AUTOMATISK_BEHANDLE,
+                false,
+            )
+
+        if (skalGjoeresAutomatisk) {
+            logger.info("Automatisk behandling: starter omregning for sak ${request.sak.sakId}")
+            startEnkeltSak(
```

Comment:
Mye logikk som skjer her vil ikke være likt for automatisk behandling av mottatte inntektsjustering. Som nevnt i forrige kommentar er det nok best med egen service